Handwriting/HandwritingIntentCSSValue.md

```diff
@@ -0,0 +1,176 @@
+* Application with custom form controls that accept sensitive input, have a strict format, or include special characters.
+* Application with custom text inputs or editing experiences that override default browser behaviors by observing and handling input events and editing experiences, doesn't support input method editor (IME) or `composition{start|end|update}` events, or if for any reason the experience designed by website authors doesn't behave as they intend when handwriting input is available.
+  
+
+![example of handwriting input inserting the text "Hello World"](./handwriting.gif "example of handwriting input inserting the text \"Hello World\"")
+
+## Proposed Solution
+
+Introduce a new value, `handwriting`, to the CSS property `touch-action` which allows authors to specify whether an element should allow handwriting input. 
+
+The `touch-action` CSS property is used by authors to define for whether user agents should execute their [direct manipulation](https://w3c.github.io/pointerevents/#dfn-direct-manipulation) behavior for touch and pen gestures. When the spec was written this only included panning and zooming. The `manipulation` value for `touch-action` will be expanded to also indicate that the user agent may consider handwriting interactions on the element.
+
+When the `touch-action` CSS property is specified for an Element, only the mentioned behaviors will be enabled on the Element and all the possible `touch-action` values that are not explicitly mentioned are then disabled for the Element.
+
+Authors are used to the [recommended practice of adding touch-action: none](https://w3c.github.io/pointerevents/#example_10) to elements over which they wish to handle all events themselves.
+
+
+### Syntax
+
+```html
+<style>
+.handwritable {
+  touch-action: handwriting;
+}
+.not-handwritable {
+  touch-action: pan-x;
+}
+</style>
+
+<textarea class="handwritable"></textarea>                                         <!-- the computed handwriting value is true -->
+<textarea class="not-handwritable"></textarea>                                         <!-- the computed handwriting value is false -->
+
+<textarea></textarea>                                         <!-- the computed handwriting value is true -->
+<textarea style="touch-action:handwriting;"></textarea>       <!-- the computed handwriting value is true -->
+<textarea style="touch-action:pan-x;"></textarea>             <!-- the computed handwriting value is false -->
+<textarea style="touch-action:pan-x handwriting;"></textarea> <!-- the computed handwriting value is true -->
+<textarea style="touch-action:manipulation;"></textarea>      <!-- the computed handwriting value is true -->
+
+
+<div style="touch-action:pan-x;">
+  <textarea></textarea>                                       <!-- the computed handwriting value is false -->
+  <textarea style="touch-action:handwriting;"></textarea>     <!-- the computed handwriting value is true -->
```

Comment:
Based on my understanding of [1] and analogy with [2] I think the computed handwriting value is false here. Similar comment for the paragraph below. @adettenb - what do you think?

[1] https://w3c.github.io/pointerevents/#determining-supported-direct-manipulation-behavior
[2] https://w3c.github.io/pointerevents/#example_10

Reply:
You're understanding is correct.
These should be false, they have an ancestor that's more restrictive.
For any particular Node, itself and all ancestors must allow `handwriting` for it to be allowed.
Thanks for catching this!

Reply:
Good catch, fixed.